proposal.rst

```diff
@@ -239,7 +264,113 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-FILLME
+Overview:
+	| First 2 weeks : Modify to use swig's -builtin function
+	| 3rd week : working on issue #263 (https://trac.xapian.org/ticket/263)
+	| 4th - 8th week : Adding support for PyPy
+	| 9th - 12th week : Adding PyPI package for Python bindings
+
+May 6 - May 27 (Community Bonding Period)
+
+	- Getting to know Xapian's codebase better
+	- Getting familiar with Swig's codebase (mainly Swig and Python part) to implement builtin
+	- Discuss my approach to solve doccomments overloading issue
+
+
+Week 1: May 27 - June 2
+
+
+	- Figure out and pass -builtin with Python inside the required Makefiles
+	- Adapt custom typemaps for excpetion handling
+	- Rectify and debug other errors (if any)
+
+
+Week 2: June 3 - June 9
+
+
+	- Test the speed and memory usage with and without passing -builtin
+	- Test whether it gives error for prewritten bindings code
+	- Documentation of changes and getting things ready to merge
+
+
+Week 3: June 10 - June 16
+
+
+	- Modify doxy2swig.py to combine the errors for all possible overloaded forms
+	- Discuss and test whether the error gets resolved by combining comments only
+	- Testing overall bindings after modification
+	- Merge the changed file
+
+
+Week 4: June 17 - June 23 and Week 5: June 24 - June 30
+
+
+	- Create module 'Xapian' to import from PyPy
+	- Create Makefiles for PyPy to get easily installed with other language bindings
+	- Write testsuite
+
+
+Week 6: July 1 - July 7
+
+
+	- Figure out and write testcases for smoketest
+	- Create smoketest file
```

Comment:
How would the smoketest for pypy be different to that for python?

Reply:
We can start with the smoketest written for python, then we can add more testcases if needed.